client/src/box/flexBox.ts

```diff
@@ -3,4 +3,11 @@ import styled from 'styled-components'
 export const Wrapper = styled.div`
     display: flex;
     align-items: center;
+`
+export const WrapperColumn = styled.div`

```

Comment:
I think it makes more sense to name `FlexColumn` instead of `WrapperColumn `

Reply:
i will do